packages/core/src/utils/__test__/memoize.test.ts

```diff
@@ -26,9 +26,9 @@ describe('memoize', () => {
     const spyComparer = jest.fn().mockReturnValue(true);
     const cachedFunc = memoize(spyFunc, spyComparer);
 
-    cachedFunc(...args);
-    cachedFunc(...args);
-    cachedFunc(...args);
+    cachedFunc(args);
```

Comment:
Question, Suggestion: Do we really need so many mock here?
If we remove mock it's make code clear:
`
    const cachedFunc = memoize(() => {}, () => true);

    cachedFunc(2, 2);
`